test/components/views/avatars/DecoratedRoomAvatar-test.tsx

```diff
@@ -66,4 +74,31 @@ describe("DecoratedRoomAvatar", () => {
 
         expect(asFragment()).toMatchSnapshot();
     });
+
+    it("shows the presence indicator in a DM room that also has functional members", async () => {
+        const DM_USER_ID = "@bob:foo.bar";
+        const dmRoomMap = {
+            getUserIdForRoomId: () => {
+                return DM_USER_ID;
+            },
+        } as unknown as DMRoomMap;
+        jest.spyOn(DMRoomMap, "shared").mockReturnValue(dmRoomMap);
+        const { container, asFragment } = render(<DecoratedRoomAvatar room={room} size="32px" />, {
+            wrapper: TooltipProvider,
+        });
```

Comment:
could you pull this out to a `renderComponent` method in the test class, so it can be shared between tests?